playbooks/group_vars/repository.yml

```diff
@@ -16,3 +16,33 @@ acs_play_repository_extra_war_downloads: []
   #   dest: "{{ content_folder }}/web-server/webapps/your-war.war"
   #   url_username: your_username_to_repo
   #   url_password: your_password_to_repo
+
+acs_play_repository_amp_community_enabled: false
+
+acs_play_repository_amp_alfresco_googledrive_repo_enterprise_artifact_name: alfresco-googledrive-repo-enterprise
+acs_play_repository_amp_alfresco_googledrive_repo_enterprise_repository: "{{ nexus_repository.enterprise_releases }}/integrations"
+acs_play_repository_amp_alfresco_googledrive_repo_enterprise_archive_url: "{{ acs_play_repository_amp_alfresco_googledrive_repo_enterprise_repository }}/{{ acs_play_repository_amp_alfresco_googledrive_repo_enterprise_artifact_name }}/{{ acs_play_repository_amp_alfresco_googledrive_repo_enterprise_version | default(omit) }}/{{ acs_play_repository_amp_alfresco_googledrive_repo_enterprise_artifact_name }}-{{ acs_play_repository_amp_alfresco_googledrive_repo_enterprise_version | default(omit) }}.amp"
+
+acs_play_repository_amp_alfresco_googledrive_repo_community_artifact_name: alfresco-googledrive-repo-community
+acs_play_repository_amp_alfresco_googledrive_repo_community_repository: "{{ nexus_repository.releases }}/integrations"
+acs_play_repository_amp_alfresco_googledrive_repo_community_archive_url: "{{ acs_play_repository_amp_alfresco_googledrive_repo_community_repository }}/{{ acs_play_repository_amp_alfresco_googledrive_repo_community_artifact_name }}/{{ acs_play_repository_amp_alfresco_googledrive_repo_community_version | default(omit) }}/{{ acs_play_repository_amp_alfresco_googledrive_repo_community_artifact_name }}-{{ acs_play_repository_amp_alfresco_googledrive_repo_community_version | default(omit) }}.amp"
```

Comment:
can't we simply override in community vars the same vars of a generic `acs_play_repository_amp_googledrive_repo_*`? we are never going to install both enterprise and community version